cmd/nerdctl/system.go

```diff
@@ -33,6 +39,40 @@ func newSystemCommand() *cobra.Command {
 	systemCommand.AddCommand(
 		newEventsCommand(),
 		newInfoCommand(),
+		newBuildPruneCommand(),
```

Comment:
This has to be `nerdctl builder prune`
https://docs.docker.com/engine/reference/commandline/builder_prune/